test/cppsuite/configs/background_compact_default.txt

```diff
@@ -0,0 +1,61 @@
+# Configuration for background_compact.
```

Comment:
There is an extra line between each category in this config and not in the stress one, can we remove the extra lines in this one?

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9691/commits/0014dfa45ae7c27ab3a958f59f8131232b05ea77